dependencies/setupFmi4c.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This entire script should be set user executable, also add it to the setupAll.sh and setupAll.bat scripts